src/main/java/org/jabref/logic/exporter/CffExporter.java

```diff
@@ -0,0 +1,227 @@
+                        data.put("date-released", localDate.toString());
+                    } catch (DateTimeParseException e) {
+                        try {
+                            formatString = "yyyy-MM";
+                            DateTimeFormatter formatter = DateTimeFormatter.ofPattern(formatString);
+                            YearMonth yearMonth = YearMonth.parse(date, formatter);
+                            int month = yearMonth.getMonth().getValue();
+                            int year = yearMonth.getYear();
+                            data.put("month", month);
+                            data.put("year", year);
+                        } catch (DateTimeParseException f) {
+                            try {
+                                formatString = "yyyy";
+                                DateTimeFormatter formatter = DateTimeFormatter.ofPattern(formatString);
+                                int year = Year.parse(date, formatter).getValue();
+                                data.put("year", year);
+                            } catch (DateTimeParseException g) {
+                                data.put("issue-date", date);
+                            }
+                        }
+                    }
+                }
+                entryMap.remove(StandardField.DATE);
+
+                // Fields
+                Map<Field, String> fieldMap = getFieldMappings();
+                for (Field field : entryMap.keySet()) {
+                    if (fieldMap.containsKey(field)) {
+                        data.put(fieldMap.get(field), entryMap.get(field));
+                    } else if (field instanceof UnknownField) {
+                        data.put(field.getName(), entryMap.get(field));
+                    }
+                }
+            }
+
+            yaml.dump(originalData, writer);
+        } catch (IOException ex) {
+            throw new SaveException(ex);
+        }
+    }
+
+    private Map<EntryType, String> getTypeMappings() {
+        Map<EntryType, String> typeMappings = new HashMap<>();
+        typeMappings.put(StandardEntryType.Article, "article");
+        typeMappings.put(StandardEntryType.Conference, "article");
+        typeMappings.put(StandardEntryType.Book, "book");
+        typeMappings.put(StandardEntryType.Booklet, "pamphlet");
+        typeMappings.put(StandardEntryType.InProceedings, "conference-paper");
+        typeMappings.put(StandardEntryType.Proceedings, "proceedings");
+        typeMappings.put(StandardEntryType.Misc, "misc");
+        typeMappings.put(StandardEntryType.Manual, "manual");
+        typeMappings.put(StandardEntryType.Report, "report");
+        typeMappings.put(StandardEntryType.TechReport, "report");
+        typeMappings.put(StandardEntryType.Unpublished, "unpublished");
+        return typeMappings;
+    }
+
+    private Map<Field, String> getFieldMappings() {
+        Map<Field, String> fieldMappings = new HashMap<>();
```

Comment:
Make map `public static final` to avoid initialization at each export.

(This also removes the method)